flume-ng-channels/flume-kafka-channel/src/test/java/org/apache/flume/channel/kafka/TestBasicFunctionality.java

```diff
@@ -49,6 +49,7 @@ public void testProps() throws Exception {
     Context context = new Context();
     context.put("kafka.producer.some-parameter", "1");
     context.put("kafka.consumer.another-parameter", "1");
+    context.put("offsets.topic.replication.factor", "1");
```

Comment:
What does this change bring? Did we need this to get the test working?

Reply:
Removed it now.  Initially, the tests failed with an error and an online search suggested to have this property in kafka-server.properties.

Reply:
The tests intermittently fail with this error.
`Number of alive brokers '1' does not meet the required replication factor '3' for the offsets topic (configured via 'offsets.topic.replication.factor'). This error can be ignored if the cluster is starting up and not all brokers are up yet`.  The tests would pass consistently by having the property `offsets.topic.replication.factor = 1` in `kafka-server.properties`.